### PR TITLE
Use RealRPC for API calls

### PR DESCRIPTION
**Pull request by cdonati**
This fixes error handling for the urlfetch API.

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2816/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2882/

File: AppServer/google/appengine/api/urlfetch_stub.py [hunk 1]
```diff
@@ -420,11 +420,21 @@ def _RetrieveURL(url, payload, method, headers, request, response,
...
+      except _fancy_urllib_InvalidCertException, e:
         raise apiproxy_errors.ApplicationError(
           urlfetch_service_pb.URLFetchServiceError.SSL_CERTIFICATE_ERROR,
           str(e))
+      except _fancy_urllib_SSLError, e:
+
+
+
+
+
```
Comment (whoarethebritons) [inline AppServer/google/appengine/api/urlfetch_stub.py]: why are there so many new lines

File: AppServer/google/appengine/api/urlfetch_stub.py [hunk 2]
```diff
@@ -420,11 +420,21 @@ def _RetrieveURL(url, payload, method, headers, request, response,
...
             socket.setdefaulttimeout(orig_timeout)
           connection.close()
-      except (_fancy_urllib_InvalidCertException,
-              _fancy_urllib_SSLError), e:
+      except _fancy_urllib_InvalidCertException, e:
         raise apiproxy_errors.ApplicationError(
           urlfetch_service_pb.URLFetchServiceError.SSL_CERTIFICATE_ERROR,
           str(e))
+      except _fancy_urllib_SSLError, e:
+
```
Comment (tmarballi) [inline AppServer/google/appengine/api/urlfetch_stub.py]: Could you remove these empty newlines?

File: AppServer/google/appengine/api/urlfetch_stub.py (hunk 1 above)
Comment (cdonati) [inline AppServer/google/appengine/api/urlfetch_stub.py]: Haha, it's the SDK code. I preserved the useless lines to reduce the diffs when comparing. It makes upgrading to new SDKs easier.

File: AppServer/google/appengine/api/urlfetch_stub.py (hunk 2 above)
Comment (tmarballi) [inline AppServer/google/appengine/api/urlfetch_stub.py]: Never mind, you just answered the question above.

File: AppServer/google/appengine/api/urlfetch_stub.py (hunk 1 above)
Comment (whoarethebritons) [inline AppServer/google/appengine/api/urlfetch_stub.py]: @cdonati 5 blank lines are needed, 5 blank lines are desired.

**Pull request merged**